src/tools/pub/versioning.md

```diff
@@ -341,8 +341,8 @@ result is a complete list of every package that your app depends on either
 directly or indirectly and the best version of that package that will work with
 your app's constraints.
 
-Pub takes that and writes it out to a **lockfile** in your app's directory
-called `pubspec.lock`. 
+Pub takes that information, computes a [content hash][] from it,
```

Comment:
This reads like it computes just one content hash of the whole list of dependencies, but doesn't it do it for each package?

Reply:
Correct, added "For each package..." to the beginning